pyproject.toml

```diff
@@ -30,15 +21,35 @@ classifiers = [
     "Programming Language :: Python :: Implementation :: CPython",
     "Programming Language :: Python :: Implementation :: PyPy",
 ]
+dynamic = ["dependencies"]
```

Comment:
education, what does this accomplish?

Reply:
However, since this project has no dependencies, we might as well specify project.dependencies = []